base/special/trig.jl

```diff
@@ -318,6 +318,60 @@ for (finv, f) in ((:sec, :cos), (:csc, :sin), (:cot, :tan),
     end
 end
 
+"""
+    cotd(x)
+
+Compute the cotangent of `x`, where `x` is in degrees.
+""" cotd
```

Comment:
Did you mean to have a space here rather than a newline? The latter would be better for consistency.